churaverse-plugins-server/src/churarenPlugin/churarenBossPlugin/domain/collisionBossDamageCause.ts

```diff
@@ -0,0 +1,19 @@
+import { Entity } from 'churaverse-engine-server'
+import { ChurarenEnemyDamageCause } from '@churaverse/churaren-core-plugin-server'
+
+/**
+ * ボスとの当たり判定によるダメージの抽象クラス
```

Comment:
抽象クラスではないので修正した方が良さそうです
```suggestion
 * ボスとの当たり判定によるダメージを表現したクラス
```